File: NXTBot/src/uk/co/shanksi/nxt/NxtCommand.java
package uk.co.shanksi.nxt;

public interface NxtCommand
{
	byte START_PROGRAM = 0x00;
	byte STOP_PROGRAM = 0x01;
	byte PLAY_SOUND_FILE = 0x02;
	byte PLAY_TONE = 0x03;
	byte SET_OUTPUT_STATE = 0x04;
	byte SET_INPUT_MODE = 0x05;
	byte GET_OUTPUT_STATE = 0x06;
	byte GET_INPUT_VALUES = 0x07;
	byte RESET_SCALED_INPUT_VALUE = 0x08;
	byte MESSAGE_WRITE = 0x09;
	byte RESET_MOTOR_POSITION = 0x0A;
	byte GET_BATTERY_LEVEL = 0x0B;
	byte STOP_SOUND_PLAYBACK = 0x0C;
	byte KEEP_ALIVE = 0x0D;
	byte LS_GET_STATUS = 0x0E;
	byte LS_WRITE = 0x0F;
	byte LS_READ = 0x10;
	byte GET_CURRENT_PROGRAM_NAME = 0x11;
	byte MESSAGE_READ = 0x13;
	
}

File: android/app/src/main/java/shanksi/legonxtapplication/nxt/NxtCommand.kt
package shanksi.legonxtapplication.nxt

object NxtCommand {
    const val START_PROGRAM: Byte = 0x00
    const val STOP_PROGRAM: Byte = 0x01
    const val PLAY_SOUND_FILE: Byte = 0x02
    const val PLAY_TONE: Byte = 0x03
    const val SET_OUTPUT_STATE: Byte = 0x04
    const val SET_INPUT_MODE: Byte = 0x05
    const val GET_OUTPUT_STATE: Byte = 0x06
    const val GET_INPUT_VALUES: Byte = 0x07
    const val RESET_SCALED_INPUT_VALUE: Byte = 0x08
    const val MESSAGE_WRITE: Byte = 0x09
    const val RESET_MOTOR_POSITION: Byte = 0x0A
    const val GET_BATTERY_LEVEL: Byte = 0x0B
    const val STOP_SOUND_PLAYBACK: Byte = 0x0C
    const val KEEP_ALIVE: Byte = 0x0D
    const val LS_GET_STATUS: Byte = 0x0E
    const val LS_WRITE: Byte = 0x0F
    const val LS_READ: Byte = 0x10
    const val GET_CURRENT_PROGRAM_NAME: Byte = 0x11
    const val MESSAGE_READ: Byte = 0x13
}